server/app/views/applicant/CreateAccountAlertFragment.html

```diff
@@ -11,13 +11,15 @@ <h2 class="usa-alert__heading" th:text="#{alert.createAccount}"></h2>
 
     <div class="content-spacing"></div>
 
-    <button
-      type="button"
-      class="usa-button usa-button--inverse"
-      th:text="#{button.createAnAccount}"
-      id="create-account"
-      th:attr="data-redirect-to=${createAccountLink}"
-    ></button>
+    <div style="display: inline-flex">
```

Comment:
**suggestion (blocking):** I agree with Gwen that we don't want to use inline styles.  Here's an interesting article that explains a few reasons why:  [avoiding inline styles](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846).  
I'd suggest taking a look at the [USWDS layout grid](https://designsystem.digital.gov/utilities/layout-grid/) to see if you could use that.  It's what we've been trying to use on the North Star to stay consistent with how we style the layout.  Then, you'll just be able to use their CSS class directly on the element.